### ISSUE TITLE
training issue: train-labels / test-labels format

### ISSUE THREAD

**Issue by DanielMolina** [andreaazzini/segnet#17]
Hello,

I'm getting the following error when I train my model after preparing my TFRecords files:
`OutOfRangeError (see above for traceback): RandomShuffleQueue '_2_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 5, current size 0) [[Node: shuffle_batch = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_FLOAT], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](shuffle_batch/random_shuffle_queue, shuffle_batch/n)]]`

All my images are 224x224 .PNGs, so I'm assuming that there's either a formatting issue with my training and test labelled images, or an issue with me using me using the CPU version of TensorFlow.

If the issue isn't from using the CPU version of TensorFlow, what sort of transforming is needed for the image annotations before using the tfrecorder to convert them? Does it involve converting the annotated images into color-mapped images using the colors from the config.py file?

Thanks!!

**Comment by MuntahaFHSTP** [comment]
may be you can reduce the batch size according to the input images

**Comment by DanielMolina** [comment]
It seems I couldn't use the annotation matrix, but once I switched to a color-mapped image, it worked.

**Comment by MuntahaFHSTP** [comment]
for me too it worked with color map......
but to view it in tensor board label and output image is different from input image while testing......its kind of confusing..... is there any way to write the output images in a directory???

**Comment by DanielMolina** [comment]
I'm not sure. TensorBoard worked for me to display both my training and testing results.

**Comment by MuntahaFHSTP** [comment]
for me in case of training its working fine....but in testing tensor board is not working as expected.  it will be very helpful if we find the way to write them in some directory out from tensor board.
